<launch>
	<node pkg="camera_wrapper" type="multi_cam" name="" output="screen">
		<param name="cam1_params" type="str" value="/home/xuchengjun/catkin_ws/src/camera_wrapper/calibration_data/003415165047" />
		<param name="cam2_params" type="str" value="/home/xuchengjun/catkin_ws/src/camera_wrapper/calibration_data/010956740447" />
		<param name="calibMode" type="bool" value="false" />
		<param name="img_scale" type="double" value="1.0" />
	</node>
	
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find human_pose)/rviz/hri.rviz" required="true" />
</launch>
